templates/python/custom-copilot-basic/src/requirements.txt

```diff
@@ -0,0 +1,2 @@
+aiohttp
+teams_ai
```

Comment:
specify a semantic version for teams-ai package

Reply:
added